src/api/txs.js

```diff
@@ -103,20 +105,47 @@ class TxController implements IController {
 
   async storeTxAsPending(tx: TxType) {
     this.logger.debug(`txs.storeTxAsPending ${JSON.stringify(tx)}`)
-    await this.storageProcessor.storeTx(tx)
+    return this.db.storeTx(tx)
+  }
+
+  async storeTxAsFailed(tx: TxType) {
+    const failedTx = {
+      ...tx,
+      status: TX_STATUS.TX_FAILED_STATUS,
+    }
+    this.logger.debug(`txs.storeTxAsFailed ${JSON.stringify(tx)}`)
+    return this.db.storeTx(failedTx, [], false)
```

Comment:
`storeTx` doesn’t seem to return anything

Reply:
`storeTx` is `async` so it returns the `Promise` of nothing )

Reply:
if it doesn’t actually return anything, it just makes it difficult to read

Reply:
makes sense

Reply:
makes sense